iib/workers/tasks/git_utils.py

```diff
@@ -181,6 +181,23 @@ def commit_and_push(
     final_commit_message = commit_message or (
         f"IIB: Update for request id {request_id} (overwrite_from_index)"
     )
+
+    log.info("Committing changes to local Git repository.")
```

Comment:
Adding this will break existing `push_configs_to_git` workflow. Consider having another helper function for this called directly from your handler?

Reply:
Extracted it to a helper function